docs/operations/logging.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Tracing in SlateDB
+---
+
+# Tracing in SlateDB
+
+SlateDB uses the [`tracing`](https://github.com/tokio-rs/tracing) library for logging and diagnostics. This provides structured, contextual logging that helps debug and monitor your database operations.
+
+## Getting Started
+
+To enable tracing in your SlateDB application, you'll need to add the following dependencies to your `Cargo.toml`:
```

Comment:
This is not quite correct. You don't need `env_logger` to enable logging. In fact, we don't include `env_logger` at all in SlateDB's dependencies and can see logs just fine using the init() command I showed earlier.

Reply:
Sounds good, I will have it updated using tracing_subscriber::fmt::init(); instead of env_logger::init().

I was following the instructions from the issue comment you referenced, which mentioned using env_logger for the documentation. 
https://github.com/slatedb/slatedb/issues/129#issuecomment-2342351820